baseapp/baseapp.go

```diff
@@ -1060,6 +1076,60 @@ func (app *BaseApp) runMsgs(ctx sdk.Context, msgs []sdk.Msg, msgsV2 []protov2.Me
 	}, nil
 }
 
+// simulateNestedMessages simulates a message nested messages.
+func (app *BaseApp) simulateNestedMessages(ctx sdk.Context, msg sdk.Msg) error {
```

Comment:
```suggestion
func (app *BaseApp) simulateNestedMessages(ctx context.Context, msg sdk.Msg) error {
```

Reply:
this calls runMsgs internally so a sdk.Context is needed